build/tzdbupdate/update-master.sh

```diff
@@ -20,9 +20,11 @@ declare -r OUTPUT=$SRCDIR/NodaTime/TimeZones/Tzdb.nzd
 declare -r DATADIR=$ROOT/data
 declare -r WWWDIR=$SRCDIR/NodaTime.Web/wwwroot
 
-dotnet restore -v quiet $SRCDIR/NodaTime.sln
-dotnet build $SRCDIR/NodaTime.sln
+echo "Building solution"
```

Comment:
nit: prefer single quotes unless you're actually relying on string interpolation.